### DISCUSSION TITLE
Does heritrix3 recrawl same url multiple times?

### DISCUSSION THREAD

**Discussion by mani-rai** [internetarchive/heritrix3#532] [Q&A] [answered]
I didn't found much on this topic in Google. Out of curiosity, does it recrawl same url multiple times?

**Comment by ato** [comment, marked as answer]
With the default configuration no, at least not within a single job launch. The most common workflow is to periodically relaunch the job, perhaps using one of the [deduplication mechanisms](https://github.com/internetarchive/heritrix3/wiki/Deduping-(Duplication-Reduction)) to avoid storing multiple identical records.

However there is a primitive [ReschedulingProcessor](https://github.com/internetarchive/heritrix3/wiki/H3-Dev-Notes-for-Crawl-Operators#fixed-rescheduling) which in theory can be configured to revisit on a fixed delay. Some users like the [UK web archive](https://anjackson.net/2018/11/13/continuous-incremental-heritrix/) have more elaborate setups for revisiting.
